### ISSUE TITLE
Play2 applications should be able to consume external Datasources

### ISSUE THREAD

**Issue by dlecan** [play2war/play2-war-plugin#13]
Setup a new Play Plugin to provide Datasources ?

**Comment by cjk** [comment]
I'm not understanding this one - does "consume external datasources" mean right now my Play2 app cannot access a Oracle- or DB2-database if I deploy it using this plugin e.g. in Websphere ?

**Comment by dlecan** [comment]
This is a "requirement", that is to say this may be already available but not specifically tested because this issue is still opened.

To be clear: I have never tested Play Framework v2.x app connected to a WAS connection pool (or JBoss one).
But no one has never reported any issue about a such configuration.
You should give it a try!

**Comment by cjk** [comment]
Ah, I see, thanks for clarifying. I'll report back about my experience once I've tried this out.

**Comment by dlecan** [comment]
Users have reported that it works, when JTA is disabled.

**Comment by poornerd** [comment]
@cjk can you summarize how you got it working? I need to get Play2 with WAS and Oracle working without JTA

**Comment by cjk** [comment]
@poornerd 
I'm afraid we no longer pursue the above combination. Requirements have since changed towards WAS+DB2+OpenJPA (without Play2 for now). I'll report back if I get a chance to try your setup again, but for now this seems rather unlikely.
